LocalPackages/PreferencesUI-macOS/Sources/PreferencesUI-macOS/View+roundedBorder.swift

```diff
@@ -1,7 +1,7 @@
 //
-//  RoundedBorder.swift
+//  View+roundedBorder.swift
```

Comment:
Could we rename it as View+RoundedBorder?

Reply:
Good catch.  Fixed.